test/unittests/commands/excel2xml/test_excel2xml_lib.py

```diff
@@ -554,6 +556,22 @@ def test_make_text_prop(self) -> None:
                 "&lt;escaped tag&gt;",
                 "&lt;escaped tag&gt;",
             ],
+            [
+                "text < text",
+                "text &lt; text",
+            ],
+            [
+                "text & text",
+                "text &amp; text",
+            ],
+            [
+                "text <unclosed> tag",
+                "text &lt;unclosed&gt; tag",
+            ],
+            [
+                'text <unclosed tag="tag"> text',
+                'text &lt;unclosed tag="tag"&gt; text',
+            ],
```

Comment:
The price that we have to pay for the current approach is, that it is possible to have invalid XML that just gets escaped. But I think the gain in usability for RDU is big enough, and the frequency of invalid XML tags low enough, to justify the approach of the current PR.

Reply:
I too think this is an improvement nonetheless